docs/pages/use-teleport/teleport-connect.mdx

```diff
@@ -0,0 +1,108 @@
+connect to a server.
+
+## Connecting to a Database
+
+1. Open a tab with cluster resources by clicking on the plus symbol at the end of the tab bar. You
+   can also press `Cmd + T` to achieve the same result.
+2. Select the Databases section.
+3. Look for the database server you wish to connect to and click the Connect button to the right.
+4. Select or enter the database user you with to use and press `Enter`.
+5. A new tab will open with a new connection established between your device and the database
+   server.
+
+This connection will remain active until you click the Close Connection button or close Teleport
+Connect. The port number will persist between app restarts – you can set up your favorite client
+without worrying about the port suddenly changing.
+
+### With a GUI Client
+
+To connect with a GUI client, follow the instructions in the database connection tab under the
+Connect with GUI section.
+
+### With a CLI Client
+
+The database connection tab shows the command that can be used to connect to the database. You can
+modify the database name of the connection and then click the Run button to open a new
+terminal tab with that command executed.
+
+## Connecting to Multiple Clusters
+
+After you add your first cluster, you can keep adding more by opening the profile selector in the
+top right and clicking at the *+ Add another cluster* button.
+
+At the moment Teleport Connect supports only one user per cluster. To log in as a different user,
+log out of the cluster first.
+
+## Restarting and Reconnecting
+
+Before closing, Teleport Connect will remember the tabs that you had open at the end of the session.
+Next time you open the app, Connect will ask you if you want to reopen those tabs. If you agree,
+Connect will restore connections to all resources that were active before you closed the app.
+
+At the moment, terminal tabs are restored only as far as the working directory is concerned. Teleport
+Connect doesn't attempt to re-execute commands that were in progress when the app was closed.
+
+## Troubleshooting
+
+In an attempt to resolve issues with a cluster you can try logging out of it, closing the app and
+logging in again.
+
+To force the app to log you out of all clusters, close the app and remove the `~/Library/Application
+Support/Teleport Connect/tsh` folder. Removing the file under `~/Library/Application
+Support/Teleport Connect/app_state.json` will clear all remembered tabs and connections.
+
+### Submitting an Issue
+
+To submit an issue, click the Submit Feedback button at the bottom right (the speech bubble symbol)
+and follow the *Submit a Bug* link.
+
+Be sure to attach logs which can be found under `~/Library/Application Support/Teleport
+Connect/logs`. The version of the app can be found in the app menu under the About Teleport Connect
+menu item.
+
+### Updating local shell environment
+
+Teleport Connect updates the environment variables on app restart and not when starting a new shell
```

Comment:
> the environment variables

Which environment variables does this refer to?

Reply:
It refers to whatever env variables are set as a result of running your shell startup files.

This should really say just "local shell environment" rather than "environment variables" specifically.